Packs/MicrosoftGraphUser/Integrations/MicrosoftGraphUser/README.md

```diff
@@ -1597,3 +1599,112 @@ For more details about the authentication used in this integration, see <a href=
 </div>
 </div>
 </div>
+<div class="cl-preview-section">
+<h3 id="user-account-disable">13. Disable a user account</h3>
+</div>
+<div class="cl-preview-section"><hr></div>
+<div class="cl-preview-section">
+<p>Disables a user from all Office 365 applications, and prevents sign in. Note: This command disables user, but does not terminate an existing session. Supported only in a self deployed app flow with the Permission: Directory.AccessAsUser.All(Delegated)</p>
+</div>
+<div class="cl-preview-section">
+<h5 id="base-command">Base Command</h5>
+</div>
+<div class="cl-preview-section">
+<p><code>msgraph-user-account-disable</code></p>
+</div>
+<div class="cl-preview-section">
+<h5 id="input">Input</h5>
+</div>
+<div class="cl-preview-section">
+<div class="table-wrapper">
+<table style="width: 749px;">
+<thead>
+<tr>
+<th style="width: 243px;"><strong>Argument Name</strong></th>
+<th style="width: 367px;"><strong>Description</strong></th>
+<th style="width: 130px;"><strong>Required</strong></th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td style="width: 243px;">user</td>
+<td style="width: 367px;">User ID or userPrincipalName.</td>
+<td style="width: 130px;">Required</td>
+</tr>
+</tbody>
+</table>
+</div>
+</div>
+<p> </p>
+<div class="cl-preview-section">
+<h5 id="context-output">Context Output</h5>
+</div>
+<div class="cl-preview-section">
+<p>There is no context output for this command.</p>
+</div>
+<div class="cl-preview-section">
+<h5 id="command-example">Command Example</h5>
+</div>
+<div class="cl-preview-section">
+<pre>msgraph-user-account-disable user="oren@demistodev.onmicrosoft.com"</pre>
+</div>
+<div class="cl-preview-section">
+<h5 id="human-readable-output">Human Readable Output</h5>
+</div>
+<div class="cl-preview-section">
+<p>user: <a href="mailto:%22oren@demistodev.onmicrosoft.com">"oren@demistodev.onmicrosoft.com</a>" account has been disabled successfully</p>
```

Comment:
please use a different account and not Oren's (change in all places here would be great).
test@demistodev.onmicrosoft.com